### DISCUSSION TITLE
Login just redirects back to /login

### DISCUSSION THREAD

**Discussion by james0r** [austintoddj/canvas#1066] [Troubleshooting]
I deployed my site to Dreamhost and ran the install commands and migrations again.

First attempt at login gave me 419 errors so I added `canvas/*` to my CSRF middle exempt array. 

Now when I attempt to login with email@example.com and password it just 302 redirects back to /login.

I verified the example user is in the DB.

**Comment by austintoddj** [comment]
Interesting. Shouldn't need to add anything to the middleware exemptions. The [CSRF tokens here](https://github.com/austintoddj/canvas/blob/master/resources/views/layout.blade.php#L7) should operate the same way as a base Laravel installation (Jetstream/Breeze).

What are you using as a session driver? (file/database)
